nh_eobs_mental_health/models/nh_eobs_ward_dashboard.py

```diff
@@ -31,7 +31,13 @@ class NHClinicalWardDashboard(orm.Model):
         'capacity_count': fields.integer('Free Beds'),
         'workload_count': fields.integer('All patients associated with ward'),
         'on_ward_count': fields.integer(
-            'All patients in bed and not on Patient Monitoring Exception')
+            'All patients in bed and not on Patient Monitoring Exception'),
+        'refused_obs_count': fields.integer(
+            'All patients currently refusing observations')
+    }
+
+    _defaults = {
+        'refused_obs_count': 0
     }
```

Comment:
But the SQL should always return a count even if it's `0` right? If that's the case this default will never be used, and if somehow it is used then that means the SQL must've not returned anything and we have no idea if `0` is actually the correct number of refused patients.